rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1073,6 +1074,10 @@ protected void doSWAPN(){
     }
 
     protected void doLOG(){
+        if (program.isStaticCall()) {
+            throw Program.ExceptionHelper.modificationException();
```

Comment:
Why this instead of just returning `new StaticCallModificationException()`?

Reply:
Consistency with the throw of other exceptions in VM.java 

I found one instance in witch we used a new and changed so it would use the ExceptionHelper.